runtime/runtime.go

```diff
@@ -182,20 +184,29 @@ func (r *runtime) Stdlib() interpreter.Importer {
 
 func (r *runtime) compilePackages() error {
 	pkgs := make(map[string]*semantic.Package)
-	for _, pkg := range r.astPkgs {
-		bs, err := json.Marshal(pkg)
+	if err := fs.WalkDir(embed.FS, "stdlib", func(path string, d fs.DirEntry, err error) error {
 		if err != nil {
 			return err
+		} else if d.IsDir() {
+			return nil
 		}
-		hdl, err := r.JSONToHandle(bs)
+		data, err := fs.ReadFile(embed.FS, path)
 		if err != nil {
 			return err
 		}
-		root, err := AnalyzePackage(hdl)
+
+		spkg, err := semantic.DeserializeFromFlatBuffer(data)
 		if err != nil {
 			return err
 		}
-		pkgs[pkg.Path] = root
+		name := strings.TrimPrefix(
+			strings.TrimSuffix(path, ".fc"),
+			"stdlib/",
+		)
+		pkgs[name] = spkg
+		return nil
+	}); err != nil {
+		return err
 	}
 	r.pkgs = pkgs
 	r.astPkgs = nil
```

Comment:
According to my reading of the code the `astPkgs` field is no longer read/used for anything. Can you remove it along with the `RegisterPackage` function? The change will break if any code is calling `RegisterPackage` so I think we should remove that function so that we are aware of any of those cases.